pkg/test/framework/components/istioctl/istioctl.go

```diff
@@ -26,12 +26,11 @@ import (
 // Instance represents "istioctl"
 type Instance interface {
 	// Invoke invokes an istioctl command and returns the output and exception.
-	// Cobra commands don't make it easy to separate stdout and stderr and the string parameter
-	// will receive both.
-	Invoke(args []string) (string, error)
+	// stdout and stderr will be returned as different strings
+	Invoke(args []string) (string, string, error)
 
 	// InvokeOrFail calls Invoke and fails tests if it returns en err
-	InvokeOrFail(t *testing.T, args []string) string
+	InvokeOrFail(t *testing.T, args []string) (string, string)
```

Comment:
This name is really odd.

FailOnInvokeError or something like that makes more sense to me